From 2561c49dcf5cb35ad72df34bf225a51fe2fa87e5 Mon Sep 17 00:00:00 2001
From: Andy Gross <agross@codeaurora.org>
Date: Thu, 12 Jun 2014 00:53:19 -0500
Subject: [PATCH 154/182] clk: qcom: Correct UTMI clock frequency table

This patch changes the UTMI clock frequency from 20MHz to the correct value of
60MHz.

Signed-off-by: Andy Gross <agross@codeaurora.org>
---
 drivers/clk/qcom/gcc-ipq806x.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/clk/qcom/gcc-ipq806x.c
+++ b/drivers/clk/qcom/gcc-ipq806x.c
@@ -1992,7 +1992,7 @@ static struct clk_branch usb30_1_branch_
 };
 
 static const struct freq_tbl clk_tbl_usb30_utmi[] = {
-	{ 60000000, P_PLL0,  1, 1, 40 },
+	{ 60000000, P_PLL8,  1, 5, 32 },
 	{ }
 };
 
